bin/pyntlm_auth/config.example.ini

```diff
@@ -0,0 +1,11 @@
+[AD]
+netbios_name=CLUSTER1
```

Comment:
Get rid of the config.example.ini file - it's not used. You'll mount the conf directory in the container and you'll access domains.conf's relevant section.